RecoEgamma/EgammaElectronProducers/python/lowPtGsfElectronSeedValueMaps_cfi.py

```diff
@@ -0,0 +1,8 @@
+import FWCore.ParameterSet.Config as cms
+
+lowPtGsfElectronSeedValueMaps = cms.EDProducer(
```

Comment:
Instead of defining a new one, you can just clone from the cfi defined in https://github.com/cms-sw/cmssw/pull/25753/files#diff-e2be6cf7e32b8b21c9d4331401b257cbR84

Reply:
Done, [here](
https://github.com/cms-sw/cmssw/pull/25753/commits/bfa006a5ab6c54c686fdff80f52cf524e4dc135c)